IOTATransferRestRequestTemplate.py

```diff
@@ -39,11 +37,8 @@ def generate_addresses(index = 0, uri='https://sandbox.iotatoken.com/api/v1/', a
     print('')
 
     # generate address based on count arg
-    addresses = []
-    for address in api.get_new_addresses(index, 1):
-        addresses.append(binary_type(address).decode('ascii'))
-    return addresses[0]
-
+    gna_result = api.get_new_addresses(index, 1)
+    return gna_result['addresses'][0]
```

Comment:
Each API method in the `Iota` class returns a dict, just to be consistent.

`get_new_addresses` always returns a dict like:

```python
{
  'addresses': [
    Address(b'...'),
    Address(b'...'),
    ...
  ]
}
```